src/test/java/xyz/vegaone/easytracking/bug/BugServiceTest.java

```diff
@@ -29,27 +31,30 @@
 
     public static final String NEWBUG_TITLE="New bug title";
 
-    public static final Long PROJECT_ID = 1L;
-
     @Autowired
     private UserStoryService userStoryService;
 
     @Autowired
     private BugService bugService;
 
-    @Test
-    public void createBugTest(){
-        //given
+    private UserStory buildAndSaveUserStory(){
```

Comment:
The method creates a UserStory object, adds data to it and then returns the object. So the name of the method should be buildUserStory(). But, but, I think you can safely save the object inside this method and then return it. This way you would keep the current name. 

Repetitive code is a pain to maintain. It's better to have all the common code in one place, so that if you want to change something, you can do that in one place instead of 4 or 10, or whatever:). Eu de ex. daca trebuie sa scriu un bloc de cod de 2 ori, il extrag imediat intr-o metoda privata.

It's a best practice. @DavidArt ^

L.E. Could you please move this method at the end of the class? When looking at a class we usually want to see the public methods first and the private methods last.